src/index.ts

```diff
@@ -13,8 +13,22 @@ export interface BaseParams {
   query?: Record<string, unknown>;
 }
 
-export const methods = ['get', 'put', 'post', 'delete', 'options', 'head', 'patch', 'trace'] as const;
```

Comment:
Deleted this. This was left over from a previous revision and no longer used.

Made the library even smaller 😎